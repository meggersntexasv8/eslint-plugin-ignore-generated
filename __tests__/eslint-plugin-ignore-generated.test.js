'use strict';

const path = require('path');
const child_process = require('child_process');

test('eslint-plugin-ignore-generated works', () => {
  const eslintRet = child_process.spawnSync(
    'node_modules/.bin/eslint',
    [
      '.',
    ],
    {
      cwd: path.join(__dirname, '__fixtures__/eslint-project'),
      encoding: 'utf8',
    }
  );
  expect(eslintRet.error).toBeUndefined();
  expect(eslintRet.stderr).toMatchSnapshot();
  expect(eslintRet.stdout).toMatchSnapshot();
  expect(eslintRet.status).toEqual(1);
});
