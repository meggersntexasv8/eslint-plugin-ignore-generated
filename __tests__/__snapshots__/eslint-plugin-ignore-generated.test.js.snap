// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`eslint-plugin-ignore-generated works 1`] = `""`;

exports[`eslint-plugin-ignore-generated works 2`] = `
"
/Users/asuarez/Downloads/eslint-plugin-ignore-generated/__tests__/__fixtures__/eslint-project/non-generated-file.js
  3:1  error  'this_variable_should_be_an_error' is not defined  no-undef

âœ– 1 problem (1 error, 0 warnings)

"
`;
